### PR TITLE
Implement table and structuredTable query types

### PR DESCRIPTION
**Pull request by bramstroker**

### CODE REVIEW

**Comment by weisswurstkanone** [comment]
Sounds cool. Could you please update the documentation as well, just mention it in the list 👍  Thx!

https://github.com/pimcore/data-hub/blob/master/doc/graphl/DataObjectQueries.md

**Comment by bramstroker** [comment]
Done

File: src/GraphQL/DataObjectQueryFieldConfigGenerator/AbstractTable.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        }
+
+        $columnKeys = $this->getTableColumnKeys($fieldDefinition);
+        if (empty($columnKeys)) {
+            return Type::string();
+        }
+
+        $fields = [];
+        foreach ($columnKeys as $key) {
+            $fields[$key] = Type::string();
```
Comment (weisswurstkanone) [inline src/GraphQL/DataObjectQueryFieldConfigGenerator/AbstractTable.php]: as you can define a type for structured table column (Number, Text, Checkbox) can we do this here as well?

Comment (bramstroker) [reply src/GraphQL/DataObjectQueryFieldConfigGenerator/AbstractTable.php]: Yes I think so. Will have a look into this wednesday

**Comment by bramstroker** [comment]
Hi, I have updated the PR to support 3 different column types for the structuredTable.
First I had used `Type:int()` for the number column, but while testing I was able to enter for example `5.5` manually in the Pimcore interface in this column, so I changed this to GraphQL type `Type:float()`.

**Comment by weisswurstkanone** [comment]
Thanks a lot!

**Pull request merged**